package.json

```diff
@@ -29,11 +29,11 @@
   },
   "peerDependencies": {
     "@emotion/core": "^10.0.28",
-    "@guardian/src-button": "2.7.1",
-    "@guardian/src-foundations": "2.7.1",
-    "@guardian/src-icons": "2.7.1",
-    "@guardian/src-link": "2.7.1",
-    "@guardian/src-text-input": "2.7.1",
+    "@guardian/src-button": "^3.3.0",
+    "@guardian/src-foundations": "^3.3.0",
+    "@guardian/src-icons": "^3.3.0",
+    "@guardian/src-link": "^3.3.0",
+    "@guardian/src-text-input": "^3.3.0",
     "@guardian/types": "^3.0.0",
     "emotion": "^10.0.27",
```

Comment:
And this?

Reply:
yes, we need to migrate each component to @emotion/react before removing (future PR)